### ISSUE TITLE
Issue with \meaning of some primitives

### ISSUE THREAD

**Issue by Jazzpirate** [brucemiller/LaTeXML#1455]
Packages like etexcmds like to check for the presence of eTeX by comparing the \meaning of primitives to their \string. More precisely, etexcmds checks that `\meaning\expanded` is equal to `\string\expanded`, and the same for `\unexpanded`, and call `\edninput` if not. 

In LaTeXML, the \meaning of both macros is `macro:#1->#1`, causing issues with etexcmds.

**Comment by dginev** [comment]
Could you point to a specific use? `\meaning` is quite difficult to emulate because the low-level internals of TeX and latexml are mostly different, so what you've described is borderline expected behavior. I've had to go and creatively find alignments between the internals of the two system for each case that has become apparent, so having this reproducible+testable would be quite helpful.

**Comment by Jazzpirate** [comment]
Easily reproducable by doing \usepackage{etexcmds} ;) Note that this will not throw an error, but rather a warning.
The check occurs in etexcmds.sty (2019/12/15 v1.7) in line 188ff

**Comment by Jazzpirate** [comment]
In general: the \meaning of any primitive (tex, etex or pdftex) is always just the macro name, and several packages happily use that to check e.g. whether etex/pdftex is used (or xetex or luatex...) before they use those primitives, or define macros differently depending on the system used.

**Comment by Jazzpirate** [comment]
Oh, if you want an actual error, `\usepackage{zref}` will do, that one will throw an actual error "Missing e-TeX's \unexpanded"

**Comment by dginev** [comment]
For future reference, the ideal issue for me would have been:
---
The tex snippet:
```tex
\message{\meaning\expanded}
\message{\meaning\unexpanded}
\message{\meaning\detokenize}

\bye
```

is needed in CTAN libraries, and and prints out:
```
$ pdftex test.tex 
This is pdfTeX, Version 3.14159265-2.6-1.40.20 (TeX Live 2019/Debian) (preloaded format=pdftex)
 restricted \write18 enabled.
entering extended mode
(./test.tex \expanded \unexpanded \detokenize )
No pages of output.
```

latexml should match that, but currently treats them as `macro:#1->#1` macros.

---
I filed one approach for a fix in #1456 .

**Comment by brucemiller** [comment]
FWIW, it's not so much that LaTeXML treats them as "macros", but that it doesn't quite distinguish between a "macro" and an expandable primitive.  Not that the distinction helps solve the issue...
